docroot/modules/humsci/hs_blocks/src/Plugin/Block/HsBackToTopBlock.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Drupal\hs_blocks\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+
+/**
+ * Provides a 'HsBackToTopBlock' block.
+ *
+ * @Block(
+ *  id = "hs_blocks_back_to_top_block",
+ *  admin_label = @Translation("Back To Top Block"),
+ * )
+ */
+class HsBackToTopBlock extends BlockBase {
+  /**
+   * {@inheritdoc}
+   */
+
+  public function build() {
+    return [
+      '#type' => 'inline_template',
+      '#template' => '<a href="#main-content" class="hs-back-to-top" hidden>Back to Top</a>',
```

Comment:
This would be better suited for a `html_tag` render array. see https://github.com/SU-HSDO/suhumsci/blob/develop/docroot/modules/humsci/hs_field_helpers/src/Plugin/Field/FieldFormatter/EntityTitleHeading.php#L83

Reply:
Also remember to translate the `Back to Top` text: `$this->t('Back to Top')`